.storybook/recipes/GlobalHeader/GlobalHeader.module.css

```diff
@@ -6,58 +6,48 @@
  * 1) 
  */
 .global-header {
+  position: relative;
   margin-bottom: var(--eds-size-4);
 
+  &:hover,
+  &:focus-within {
+    @media all and (min-width: $eds-bp-lg) {
+      width: var(--eds-l-sidebar-width);
+      overflow: visible;
+    }
+  }
+
   @media all and (min-width: $eds-bp-lg) {
-    margin-bottom: 0;
+    position: fixed;
     display: flex;
     flex-direction: column;
-    max-width: var(--eds-l-sidebar-width);
+    align-items: flex-start;
+    overflow: hidden;
+    margin-bottom: 0;
+    width: 3.75rem;
+    transition: width var(--eds-anim-move-quick) var(--eds-anim-ease);
```

Comment:
Prefers reduced motion again. Also, should we be concerned about the performance of transitioning the `width` property?